<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterSearch</class>
 <widget class="QWidget" name="CharacterSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vLayoutBase">
   <item>
    <layout class="QVBoxLayout" name="vLayoutMain">
     <item>
      <widget class="QGroupBox" name="groupSearch">
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
       <layout class="QHBoxLayout" name="hLayoutGroupSearch">
        <item>
         <widget class="QLineEdit" name="leCharacterName">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbSearch">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>검색</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupCharacter">
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <layout class="QHBoxLayout" name="hLayoutGroupCharacter">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="vLayoutProfile">
          <item>
           <widget class="QPushButton" name="pbOthers">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>보유 캐릭터</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabCharacter">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>장비</string>
           </attribute>
           <layout class="QHBoxLayout" name="hLayoutTabCharacter">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>759</width>
                 <height>527</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="hLayoutCharacterScroll">
                <property name="spacing">
                 <number>1</number>
                </property>
                <item>
                 <layout class="QVBoxLayout" name="vLayoutEquip">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="vLayoutAccessory">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>스킬</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QScrollArea" name="scrollArea_2">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_2">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>759</width>
                 <height>527</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
